lighthouse-cli/cli-flags.js

```diff
@@ -12,19 +12,32 @@ const pkg = require('../package.json');
 const printer = require('./printer');
 
 /**
- * @param {string=} manualArgv
+ * @param {(string | string[])[]} arr
+ * @return {string[]}
```

Comment:
isn't this simpler, just
```js
@param {Array<Array<string>>}
@return {Array<string>}
```
(`str.split(',')` always returns an array, even if the substring isn't found)